app/src/components/widgets/charts/launchesDurationChart/launchesDurationChart.scss

```diff
@@ -7,6 +7,10 @@
         .c3-axis-y-label {
           font-size: 12px;
         }
+        path {
```

Comment:
Maybe it would be better if we indicate which `path` we should highlight? F.e. by `.domain` selector